### PR TITLE
Custom cyanide pill suicide

### PR DESCRIPTION
**Pull request by DeityLink**
:cl:
* tweak: Suiciding while holding one of the cyanide pills that Nuke Ops spawn with now instantly deals you all the damage that it would have done to you over time. That's 500 toxloss and 500 oxyloss, so basically an instant death without having to suicide AND succumb.

### CODE REVIEW

**Comment by Eneocho** [comment]
Can it have the game over sound from MGS3 when you use the fake death pill?

**Comment by DeityLink** [comment]
This one is all BUT a fake death

File: code/modules/reagents/reagent_containers/pill.dm [hunk 1]
```diff
@@ -150,6 +150,15 @@
 	..()
 	reagents.add_reagent(CYANIDE, 50)
 
+/obj/item/weapon/reagent_containers/pill/suicide_act(mob/user)
```
Comment (SonixApache) [inline code/modules/reagents/reagent_containers/pill.dm]: missing a subtype here aren't you?

**Review by SonixApache — CHANGES_REQUESTED** [review]

File: code/modules/reagents/reagent_containers/pill.dm (hunk 1 above)
Comment (DeityLink) [reply code/modules/reagents/reagent_containers/pill.dm]: woops
```suggestion
/obj/item/weapon/reagent_containers/pill/cyanide/suicide_act(mob/user)
```

Comment (DeityLink) [reply code/modules/reagents/reagent_containers/pill.dm:153]: fixed type path

**Pull request merged**